kubernetes-jenkins-plugin/README.md

```diff
@@ -40,6 +40,32 @@ Create the pipeline (buildconfig with pipeline strategy) and start a build:
 
 ```
 $ oc create -f kube-fcos-pipeline.yaml
+```
+
+Update the "secret" token values in the webooks to be unique
+
+``` 
+$ oc set triggers bc/kubernetes-fcos-pipeline --from-github
+$ oc set triggers bc/kubernetes-fcos-pipeline --from-webhook
+```
+
+Grab the URLs of the webhooks from `oc describe` and set up webhook
+in github.
+
+- `oc describe bc/kube-fcos-pipeline`
+- From the GitHub web console.
+- In your GitHub repository, select Add Webhook from Settings → Webhooks
+  & Services.
+- Paste the URL output (similar to above) into the Payload URL field.
```

Comment:
I haven't played with this much, what is the `(similar to above)` actually referencing?

Reply:
ahh, I copy/pasted the instructions from somewhere else.. will clarify

Reply:
uploaded an attempt at fixing the problem